requirements.txt

```diff
@@ -5,4 +5,8 @@ networkx>=2.4
 pytest
 pandas>=1.0.0
 polars>=0.16.6
-pre-commit
\ No newline at end of file
+pre-commit
+kloppy~=3.11.0
```

Comment:
Kloppy shouldn't be a requirement of Kloppy.

Reply:
Not sure how I introduced these...

Deleted all requirements not present in current master, namely:
kloppy~=3.11.0
pytz~=2023.3
python-dateutil~=2.8.2
setuptools~=67.8.0

Please double check whether this is correct.

Reply:
Looks correct now. Thanks!